webserver/__init__.py

```diff
@@ -1,68 +1,83 @@
 import os
 import sys
+
+from flask.config import Config
+
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+basepath = os.path.dirname(os.path.realpath(__file__))
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 import logging
 import logging.handlers as handlers
 
-from helpers.StreckennetzSteffi import StreckennetzSteffi
-from database.engine import DB_CONNECT_STRING
+# Create app with changed paths  https://stackoverflow.com/a/42791810
+app = Flask(__name__, instance_relative_config=True,
+        template_folder='website', static_folder='website/', static_url_path='')
 
-basepath = os.path.dirname(os.path.realpath(__file__))
-
-streckennetz = StreckennetzSteffi(prefer_cache=True)
-from webserver.predictor import Predictor
-pred = Predictor()
-db = SQLAlchemy()
+logHandler = handlers.TimedRotatingFileHandler(basepath + '/logs/website.log', when='midnight', backupCount=100)
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+logHandler.setFormatter(formatter)
+logHandler.setLevel(logging.INFO)
+app.logger.addHandler(logHandler)
+app.logger.setLevel(logging.INFO)
+logger = app.logger
 
+#Print our cool logo
+from helpers.fancy_print_tcp import TCP_SIGN
 
-def create_app():
-    #Print our cool logo
-    from helpers.fancy_print_tcp import TCP_SIGN
+app.logger.info("Loading config...")
+if os.path.isfile("/mnt/config/config.py"):
+    sys.path.append("/mnt/config/")
+from config import config
```

Comment:
This makes me unable to run the website locally and btw, we don't even use the config.

Reply:
And the rest of the web server needs the config in the central file anyways.

Reply:
äh sorry?
It only uses the config from the external path if the external path exists...
You can still run the website locally with no problem what so ever...
There are even 3 ways:
* `export FLASK_APP="webserver"; flask run`
* `gunicorn "webserver:app"` _-b 0.0.0.0:5000_
* `docker build -f webserver/Dockerfile.webserver . -t webserver; `
  `docker run -p 5000:5000 -v $(pwd)/config.py:/mnt/config/config.py webserver`
  
> btw, we don't even use the config.

Yea we connect to the db?
Use the flask enviroments?

Reply:
Yes but for the db connection we are using a separate import in `database/engine.py`

Reply:
Yes that makes this line the more important,
 since it shows the programm another option where the config is...
and since the path is global this affects the other stuff too...

Reply:
Btw just sayin' this code is live...